### PR TITLE
Add VSLAM proposal

### PR DESCRIPTION
**Pull request by jike5**
Signed-off-by: jike5 <2719825969@qq.com>

## Description

* Add proposal for Implementation of an VSLAM Algorithm on KubeEdge project

* The design architecture and technical details are included in the proposal

## Related Issues

https://github.com/kubeedge/examples/issues/120

### CODE REVIEW

File: sig-robotics/proposal/Implement-VSLAM-algorithm-based-on-KubeEdge/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+### 1.1 Goals
+
+- The edge applications will be required to implement data collection and feature extraction in the simulation environment, send the data to cloud server for map creation.
+- The edge applications will be required to keep track for a  moment when the network is disconnected.
+- The cloud applications will be required to perform loop closing and global mapping tasks.
+
+## 2 Proposal
+
+We propose using one of the classic SLAM frameworks: ORB-SLAM2 as the basis. Just like ORB-SLAM2 or other classic frameworks, the SLAM system mainly includes four modules: tracking, local optimization, loop closure detection and global optimization.
```
Comment (JoeyHwong-gk) [inline sig-robotics/proposal/Implement-VSLAM-algorithm-based-on-KubeEdge/README.md:15]: Suggest: adding links to your referenced technologies

File: sig-robotics/proposal/Implement-VSLAM-algorithm-based-on-KubeEdge/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+* Only the **tracking module is deployed at the edge**, the back-end optimization and loop closing tasks are completed in the cloud, and the cloud continuously pushes local maps to the edge.
+
+The first strategy ensures that the edge has a long-term offline autonomy capability, but increases the computing and memory resource occupation of the edge. On the other hand, how to synchronize local map in the edge and global map in the cloud becomes a new problem.
+
+The second method offloads all optimizations to the cloud, which greatly reduces the pressure on edge resources, but requires strong network stability. At the same time, the edge side only needs to maintain a small-scale local map (below 10 frames), and the local map can be updated easily by clearing local map on the edge and repalcing it with new received local map update from the cloud.
+
+Based on the above discussion, based on the reliable network communication provided by kubeedge and edgemesh and some open source work available for reference(such as [edge-slam][https://github.com/droneslab/edgeslam]), we intend to use the second method for this work.
```
Comment (JoeyHwong-gk) [inline sig-robotics/proposal/Implement-VSLAM-algorithm-based-on-KubeEdge/README.md:42]: Replace: `[edge-slam][https://github.com/droneslab/edgeslam]` to `[edge-slam](https://github.com/droneslab/edgeslam)`

File: sig-robotics/proposal/Implement-VSLAM-algorithm-based-on-KubeEdge/README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+**Local Map**
+
+Local map is a subset of the global map, it contains several latest optimized key-frames and map points associated with.
+
+**Map Synchronization**
+
+To maintain the synchronization between local map and global map, the cloud      will send the latest subset of global map to edge frequently. If the update is **redundant** or **latency**, the edge will not use it.
+
+## 4 Workers Communication
```
Comment (JoeyHwong-gk) [inline sig-robotics/proposal/Implement-VSLAM-algorithm-based-on-KubeEdge/README.md:84]: If there is no more information, delete it.

**Comment by JoeyHwong-gk** [comment]
Nice job!

`Modules` recommended to be added:

**Use Cases**: Case studies are powerfully informative tools that can help you win bids with your proposals. The truth is, case studies are handy because your audience may be critical of overt claims, and the best way to dispel that doubt is to back yourself up with solid data. Case studies are valuable resources, but they must be carefully implemented to be totally effective, particularly within a convincing proposal.

File: sig-robotics/proposal/Implement-VSLAM-algorithm-based-on-KubeEdge/README.md (hunk 3 above)
Comment (jike5) [reply sig-robotics/proposal/Implement-VSLAM-algorithm-based-on-KubeEdge/README.md:84]: Got it!

Comment (jike5) [reply sig-robotics/proposal/Implement-VSLAM-algorithm-based-on-KubeEdge/README.md:84]: > If there is no more information, delete it.

Got it!

**Comment by JoeyHwong-gk** [comment]
/lgtm

& 

/ping @fisherxu

**Comment by jike5** [comment]
The proposal will be added in #109.

**Pull request closed without merging**